codec-http/src/test/java/io/netty/handler/codec/http/multipart/HttpPostRequestDecoderTest.java

```diff
@@ -491,4 +492,136 @@ public void testFormEncodeIncorrect() throws Exception {
             content.release();
         }
     }
+
+    // https://github.com/netty/netty/pull/7265
+    @Test
+    public void testDecodeContentDispositionFieldParameters() throws Exception {
+
+        final String boundary = "74e78d11b0214bdcbc2f86491eeb4902";
+
+        String encoding = "utf-8";
+        String filename = "attached_файл.txt";
+        String filenameEncoded = URLEncoder.encode(filename, encoding);
+
+        final String body = "--" + boundary + "\r\n" +
+          "Content-Disposition: form-data; name=\"file\"; filename*=" + encoding + "''" + filenameEncoded + "\r\n" +
+          "\r\n" +
+          "foo\r\n" +
+          "\r\n" +
+          "--" + boundary + "--";
+
+        final DefaultFullHttpRequest req = new DefaultFullHttpRequest(HttpVersion.HTTP_1_1,
+                                                                      HttpMethod.POST,
+                                                                      "http://localhost",
+                                                                      Unpooled.wrappedBuffer(body.getBytes()));
```

Comment:
@normanmaurer do your tests require you to deref all buffers?

Reply:
yep I think we should call req.release()

Reply:
@dminkovsky please address this

Reply:
@normanmaurer done. Please have a look. Thank you.